images/win/scripts/Installers/Post-Deployment.ps1

```diff
@@ -0,0 +1,19 @@
+# Create shells folder
+$shellPath = "C:\shells"
+$null = New-Item -Path $shellPath -ItemType Directory
+
+# MSYS2 <--> C:\msys64\usr\bin\bash.exe
+$null = New-Item -ItemType SymbolicLink -Path "$shellPath\msys2.exe" -Target "C:\msys64\usr\bin\bash.exe"
+
+# sh and bash <--> C:\msys64\usr\bin\bash.exe
+$null = New-Item -ItemType SymbolicLink -Path "$shellPath\bash.exe" -Target "C:\msys64\usr\bin\bash.exe"

```

Comment:
nitpick: May be `$shellPath\bash.exe` symlink should point to `$shellPath\msys2.exe`? Let's put this line after creation all other symlinks